blog/2023-02-15-netdata-functions.md

```diff
@@ -0,0 +1,72 @@
+---
+slug: netdata-functions
+title: Introducing Netdata Functions
+description: Introducing Netdata Functions
+authors: shyam
+tags: [real-time, monitor, troubleshoot, functions, top, processes]
+keywords: [real-time, monitor, troubleshoot, functions, top, processes]
+image: https://user-images.githubusercontent.com/24860547/219148788-393d42d3-ef34-43bd-bb8b-e612246923c5.png
+
+---
+
+Netdata is committed to making it simpler and easier for everyone to monitor and troubleshoot their infrastructure. With that goal in mind, we're excited to announce the launch of our new "Functions" feature, which allows Netdata Agent collectors to expose "functions" that can be executed in run-time and on-demand.
+
+<!--truncate-->
+
+### What are Netdata functions?
+
+Netdata has always been synonymous with real time monitoring and automated dashboards, with the recent introduction of "functions", there's now a new way for users to troubleshoot their infrastructure. **A function, in the context of Netdata, is a routine or script that can be invoked to run on a node and retrieve useful information, which is then displayed in the Netdata cloud dashboard.**
+
+Think of it as a toolkit that allows you to quickly and easily retrieve specific information about your infrastructure, without having to SSH into each node individually and run commands to retrieve that information. With functions, you can gather more information and perform actions on a given node without having to access it directly. This can save SysAdmins and DevOps engineers time and effort, especially when managing large clusters of nodes.
```

Comment:
```suggestion
Think of it as a toolkit that allows you to quickly and easily retrieve specific information about your infrastructure, without having to SSH into each node individually and run commands to retrieve that information. With functions, you can gather more information and perform actions on a given node without context switching and without having to access it directly. This can save SysAdmins and DevOps engineers time and effort, especially when managing large clusters of nodes.
```